web-module/src/main/webapp/templates/celAjax/struct/autocomplete/search.vm

```diff
@@ -2,16 +2,21 @@
 $jsonBuilder.openDictionary()##
 #set($autoComplete = $services.structuredDataEditor.getSelectTagAutoCompleteImpl($request.type).orElse($null))##
 #if($services.rightsAccess.isLoggedIn() && "$!autoComplete" != '')##
-  $jsonBuilder.addProperty('type', "$!autoComplete.name")##
-  $jsonBuilder.addProperty('cellRef', "$!request.cellRef")##
-  $jsonBuilder.addProperty('searchterm', "$!request.searchterm")##
-  #set($offset = $util.parseInt("$!request.offset"))##
+  $jsonBuilder.addProperty('type', "$!{autoComplete.name}")##
+  $jsonBuilder.addProperty('cellRef', "$!{request.cellRef}")##
+  $jsonBuilder.addProperty('searchterm', "$!{request.searchterm}")##
+  #set($cellDocRef = $services.model.resolveDocument($request.cellRef))##
+  #set($addNewUrl = "")
+  #if(${autoComplete.getUrlToNewElementEditor($cellDocRef).isPresent()})
+    #set($addNewUrl = "$!{autoComplete.getUrlToNewElementEditor($cellDocRef).get().toExternalForm()}")
+  #end
```

Comment:
now we have inconsistent naming here `urlToNewElementEditor` -> `addNewUrl`. I'd prefer it to be consistent, even if I don't like the perceived redundancy withing `addNew`. perhaps simply `createUrl`?